packages/common/src/schemas/projects/project.schema.ts

```diff
@@ -64,7 +64,11 @@ export const projectSchema = Type.Object(
     commitSHA: Type.Optional(Type.String()),
     commitDate: Type.Optional(Type.String({ format: 'date-time' })),
     assetsOnly: Type.Boolean(),
+    isPublishedVersion: Type.Boolean(),
```

Comment:
what is the use of this? what if a project has `projectPublishId` then its published otherwise its not?